packages/web-app/src/components/appli/Massif/Massif.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { isNil, isEmpty } from 'ramda';
+import Skeleton from '@material-ui/lab/Skeleton';
+import { useIntl } from 'react-intl';
+import CreateIcon from '@material-ui/icons/Create';
+
+import { Box, IconButton } from '@material-ui/core';
+import Layout from '../../common/Layouts/Fixed/FixedContent';
+import CavesList from '../../common/cave/CavesList';
+import EntrancesList from '../../common/entrance/EntrancesList';
+import DocumentsList from '../../common/DocumentsList/DocumentsList';
+import Alert from '../../common/Alert';
+import MassifPropTypes from './propTypes';
+import MapMassif from './MapMassif';
+import Descriptions from './Descriptions';
+
+const Massif = ({ isFetching, error, massif, canEdit, onEdit }) => {
+  const { formatMessage } = useIntl();
+
+  let title = '';
+  if (massif?.name) {
+    title = massif.name;
+  } else if (!error) {
+    title = formatMessage({ id: 'Loading massif data...' });
+  }
+
+  return (
+    <Layout
+      title={title}
+      subheader={
+        <>
+          {isFetching && !error ? (
+            <Skeleton />
+          ) : (
+            massif &&
+            `${formatMessage({ id: 'Language' })} : ${
+              massif?.names[0].language
+            }`
+          )}
+        </>
+      }
+      content={
+        <>
+          {isFetching && isNil(error) && (
+            <>
+              <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+                <Skeleton height={300} width={800} /> {/* Map Skeleton */}
+              </Box>
+              <Skeleton height={100} /> {/* EntranceList data Skeleton */}
+              <Skeleton height={100} /> {/* CavesList data Skeleton */}
+              <Skeleton height={100} /> {/* Documents Skeleton */}
+            </>
+          )}
+          {error && (
+            <Alert
+              title={formatMessage({
+                id:
+                  'Error, the massif data you are looking for is not available.'
+              })}
+              severity="error"
+            />
+          )}
+          {massif && (
+            <>
+              <Box
+                alignItems="start"
+                display="flex"
+                flexBasis="300px"
+                justifyContent="space-between">
+                {!isEmpty(massif.descriptions) ? (
+                  <Descriptions descriptions={massif.descriptions} />
+                ) : (
+                  <Alert
+                    severity="info"
+                    title={formatMessage({
+                      id: 'This massif does not have a description'
```

Comment:
'This massif has no description listed yet.' (histoire d'être homogène avec les autres messages)

Aussi, il manque un point à la fin.